src/HealthChecks.ClickHouse/HealthChecks.ClickHouse.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.1;$(DefaultNetCoreApp)</TargetFrameworks>
+    <PackageTags>$(PackageTags);Beat;ClickHouse</PackageTags>
+    <Description>HealthChecks.ClickHouse is a health check for ClickHouse.</Description>
+    <VersionPrefix>$(HealthCheckClickHouse)</VersionPrefix>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="ClickHouse.Client" />
+    <PackageReference Include="Microsoft.Extensions.Diagnostics.HealthChecks" />
```

Comment:
this should fix the build (it's now being defined in the `.props` file now)

https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/blob/60404686d08ae49a64f8e1b704ab49013307554b/src/Directory.Build.props#L37-L39



```suggestion
```